folium/features.py

```diff
@@ -10,22 +10,28 @@
 import json
 import warnings
 
-import requests
-import numpy as np
-
 from branca.colormap import LinearColormap, StepColormap
 from branca.element import (Element, Figure, JavascriptLink, MacroElement)
-from branca.utilities import (_locations_tolist, _parse_size, image_to_url,
-                              none_max, none_min, color_brewer)
+from branca.utilities import color_brewer
 
 from folium.folium import Map
 from folium.map import (FeatureGroup, Icon, Layer, Marker, Tooltip)
-
-from folium.utilities import get_bounds
+from folium.utilities import (
+    _iter_tolist,
+    _parse_size,
+    get_bounds,
+    image_to_url,
+    none_max,
+    none_min,
+)
 from folium.vector_layers import PolyLine
 
 from jinja2 import Template
 
+import numpy as np
+
+import requests
```

Comment:
What's the import order in folium? First standard library, then branca, then folium, then Jinja, then other packages?
Also, empty lines between other packages?

Reply:
Yep. Enforced by `flake8-import-order` (listed in the `requirements-dev.txt`). We used to enforced that in the tests and I turned it off when I added Stickler-CI. (I'm looking into another solution like Stickler but that will allow us to use all the flake8 plugins.)

Reply:
I haven't used that plugin before, but seems sensible. But seems weird to me that folium comes before jinja, requests and numpy. Since folium is the local package I think it should come last.

> In general stdlib comes first, then 3rd party, then local packages

https://github.com/PyCQA/flake8-import-order#flake8-import-order

Reply:
You are correct. My editor does that automatically and, b/c we are not using relative imports, it does not know that `folium ` is local. Let's leave it be for now. I'll see if I can fix my editor to avoid triggering false positives and I'll fix those in another PR. (Sorry for the confusion.)

Reply:
> I'll fix those in another PR

alright!